migrations/2022-08-02-041148_create_webhooks/up.sql

```diff
@@ -0,0 +1,15 @@
+-- Your SQL goes here
+CREATE TABLE webhooks (
+    id BIGSERIAL PRIMARY KEY,
```

Comment:
I don't think we use this elsewhere in the codebase yet, but I [recently learned](https://stackoverflow.com/a/55300741/2353071) of of `GENERATED ALWAYS AS IDENTITY` which can replace `SERIAL` and ensures that the app cannot generate its own id numbers (that would later conflict with the serial). This would become:

```suggestion
    id BIGINT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
```